src/app/work-item/work-item-list/work-item-list.component.ts

```diff
@@ -175,14 +175,39 @@ export class WorkItemListComponent implements OnInit, AfterViewInit {
   }
 
   // event handlers
+  onToggle(entryComponent: WorkItemListEntryComponent, event: MouseEvent): void {
```

Comment:
If onToggle accepts two arguments, in **work-item-list.component.html - 95** , why is it being called with one?